packages/mdc-button/mdc-button.scss

```diff
@@ -17,62 +17,32 @@
 @import "@material/elevation/mixins";
 @import "@material/ripple/mixins";
 @import "@material/theme/mixins";
+@import "@material/theme/variables";
 @import "@material/typography/mixins";
 
 // postcss-bem-linter: define button
-
 .mdc-button {
   @include mdc-ripple-base;
-  @include mdc-ripple-bg((pseudo: "::before"));
-  @include mdc-ripple-fg((pseudo: "::after"));
+  @include mdc-ripple-bg((pseudo: "::before", base-color: #621982, opacity: .16));
```

Comment:
I planned to do that as part of #991 , but nvm, since you already ask, I will make the change here and it will make parameterization later easier